CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Features
+
+- EF Core performance monitoring ([#1112](https://github.com/getsentry/sentry-dotnet/pull/1112))
```

Comment:
- EF Core performance monitoring ([#1112](https://github.com/getsentry/sentry-dotnet/pull/1112))

is not part of this PR but I suppose it'll be sorted out after merging the main.

Reply:
```suggestion
```